### DISCUSSION TITLE
How to trigger release.yml?

### DISCUSSION THREAD

**Discussion by hiensumi** [cpeditor/cpeditor#1287] [Q&A]
How to trigger release.yml?

**Comment by ouuan** [comment]
Push a Git tag like `0.1.2`.

**Reply by hiensumi** [reply]
![image](https://github.com/cpeditor/cpeditor/assets/47080203/5f275c9b-99f1-49a9-837d-13025e4d6d08)
Sometime it works but sometime not

**Reply by hiensumi** [reply]
![image](https://github.com/cpeditor/cpeditor/assets/47080203/6639e93a-46ee-4408-881c-05ae62bff23e)

10 commits but only 4 releases

**Reply by ouuan** [reply]
Tags, not commits, trigger it.

**Reply by hiensumi** [reply]
Ok. I did it.

**Comment by hiensumi** [comment]
![image](https://github.com/cpeditor/cpeditor/assets/47080203/431ba584-20b3-435d-bc9b-63811a4858f5)
Error with windows runner :(

**Reply by hiensumi** [reply]
I use github desktop  to push tag

**Reply by ouuan** [reply]
https://github.com/hiensumi/cpeditor/blame/4cb0284138342a1ce5b8148a8ecaf44660ae69e0/.github/workflows/release.yml#L376

https://github.com/hiensumi/cpeditor/commit/38175d3fc656b17ce2b196a0699d050188d325d8#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R376

Please check carefully before commiting/asking.

**Reply by hiensumi** [reply]
My bad omg :vvvvvvvv

**Reply by hiensumi** [reply]
Tysm :)) I know I asked too much because im new to these things. Sorry!
